build-tools/eslint/prefer-live-region.js

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+module.exports = {
+  meta: {
+    type: 'problem',
+    messages: {
+      'prefer-live-region': 'JSX element uses aria-live property. Prefer using LiveRegion component instead.',
+    },
+    docs: {
+      description: 'prevents direct use of aria-live to avoid a known issue with NVDA.',
```

Comment:
```suggestion
      description: 'prevents direct use of aria-live attributes to avoid a known issue with NVDA (described in the internal LiveRegion component).',
```

Could you elaborate in the rule's description a bit more about the known issue? As first time consumer I'd ask myself "What is this known issue about?" Would be great to get a pointer to get that answered.